internal/notification/providers/email/message.go

```diff
@@ -0,0 +1,47 @@
+package email
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+var (
+	isHTMLRgx = regexp.MustCompile(`.*<html.*>.*`)
+	lineBreak = "\n"
```

Comment:
RFC (https://tools.ietf.org/html/rfc2822#section-2.1) states to use CR LF (\r\n)